FranckKe/mersea
# [master] Refactor i18n

Use browser language setting, fallback on english
Remove unused backend i18n detection
Remove default/fallback env i18n settings
Update interface with user language on login
Fix language switcher default value

## To test:
1. Detect user browser language settings: 
- Disabled or remove application localstorage state
- Set your browser language to `es-*` or `fr-*`
- reload
- Check if initialized language is browser language
___
2. Detect fallback if user browser language is not supported: 
- Disabled or remove application localstorage state
- Set your browser language to anything else than `en-*` `es-*` `fr-*` 
- reload
- Check if initialized language is english
___
3. Language updating on login 
- Login
- Set language in settings
- Logout
- Set language in nav dropdown to a different one than the user setting
- Login
- Language should have been updated with the user language
